tests/utils.py

```diff
@@ -168,7 +162,7 @@ def __init__(self, log, server='simple_server.py',
     started = False
     port_generation_attempts = 5
     while not started and port_generation_attempts > 0:
-      self.port = port or random.randint(30000, 45000)
```

Comment:
Nit: the commit message does not tell us _why_ the argument was removed. Presumably because none of the tests are using it?